app/models/end_product_establishment.rb

```diff
@@ -215,6 +213,14 @@ def generate_tracked_item!
     end
   end
 
+  def request_issues
+    source.request_issues.select { |ri| ri.end_product_establishment == self }
+  end
+
+  def active_request_issues
+    request_issues.select { |request_issue| request_issue.removed_at.nil? }
```

Comment:
i don't think this is the test for `active`. Which makes me think we want to use a method here:

```suggestion
    request_issues.select(&:status_active?)
```

Reply:
I agree that "active" and "not removed" are different things. I want to make sure the existing logic is respected though, since the name "active" is specifically attached to "not removed" and the intent may just be mis-named.

For *this* PR though, we do want `status_active?` and I will use that.

Reply:
yeah we need to figure out the wording there. I'm thinking:

active = not resolved and not removed
complete = has decision and is "finalized"

i don't have a good word for "not removed"

Reply:
What do you mean by "not resolved"? That `status_active?` is true?

```diff
@@ -218,7 +221,7 @@ class EndProductEstablishment < ApplicationRecord
   end

   def active_request_issues
-    request_issues.select { |request_issue| request_issue.removed_at.nil? }
+    request_issues.select { |request_issue| request_issue.removed_at.nil? && request_issue.status_active? }
   end
```